File: lab2/calc.cpp
#include <iostream>
#include <chrono>

double calculate (double x) {
    return x*x - x*x + x*4 - x*5 + x + x;
}



int main()

{
    long long n;
    double res, time;
    int x = 10;

    while(1)
    {
        if (!(std::cin >> n)) 
            break; 
        
        clock_t start = clock();

        for (long long i =0; i<n; ++i)
        {
            res = calculate(x);
        }

        clock_t end = clock();
        time = (double)(end - start) / CLOCKS_PER_SEC;

        std::cout <<"time =  "<< time <<";  "<<"result(x=10) =  "<< res << "\n";
    }
}

File: lab2/calc.py
import time

def calculate(x):

    return x**2 - x**2 + x*4 - x*5 + x + x

def main():
    x = 10
    
    while True:
        user_input = input()

        if not user_input.isdigit():
            break
        
        n = int(user_input)
        
        start_time = time.time()
 
        res = 0

        for i in range(n):
            res = calculate(x)
            
        end_time = time.time()
        
        duration = end_time - start_time
        
        print(f"time = {duration} s; result(x=10) = {res}")

if __name__ == "__main__":
    main()